plugins/containers-plugin/src/containers-tree-data-provider.ts

```diff
@@ -106,7 +108,7 @@ export class ContainersTreeDataProvider implements theia.TreeDataProvider<ITreeN
                         name: commandName,
                         tooltip: 'execute the command',
                         iconPath: 'fa-cogs medium-yellow',
-                        command: { id: 'task:run', arguments: ['che', commandName] }
+                        command: { id: 'task:run', arguments: [CHE_TASK_TYPE, commandName, this.overrideContainerName(container.name)] }
```

Comment:
hello, it couldn't be another field ? looks like we're now having complex types for command

Reply:
I would not use dedicated method and constant.
Just inline
```
{
    type: 'che',
    target: {
        machineName: containerName
    }
}
```

Reply:
@vitaliy-guliy, I think @mmorhun can not do that in a simple way. To do this stuff we need align this field in the che-machine-exec server side. @vitaliy-guliy create separated issue for that.